File: function.dart
// function.dart
import 'dart:io';

int myFunction(int param) {
    int x = param;
    x += 10;
    if (x > 15) {
        x -= 5;
    } else {
        x += 5;
    }
    for (int i = 0; i < 3; i++) {
        x += i;
    }
    return x;
}

void main() {
    stdout.write("Enter an integer: ");
    String? input = stdin.readLineSync();
    int? param = int.tryParse(input!);
    if (param == null) {
        print("Invalid input");
    } else {
        print(myFunction(param));
    }
}

File: function.r
# function.r
my_function <- function(param) {
    x <- param
    x <- x + 10
    if (x > 15) {
        x <- x - 5
    } else {
        x <- x + 5
    }
    for (i in 0:2) {
        x <- x + i
    }
    return(x)
}

cat("Enter an integer: ")
input <- readLines("stdin", n=1)
param <- as.integer(input)

if (is.na(param) && input != "0") {
    cat("Invalid input\n")
} else {
    cat(my_function(param), "\n")
}
